src/loaddb/load_error_handler.cpp

```diff
@@ -45,7 +45,17 @@ namespace cubload
 #endif
 
   int
-  error_handler::get_lineno ()
+  error_handler::get_driver_lineno ()
```

Comment:
@lucianlazar2211 

what do you think about setting the scanner starting line offsetted by 1 from the start?

in invoke_parser:

  int parser_result = driver->parse (iss, batch_.get_line_offset () + 1);

Then we won't have to add +1 in `get_scanner_lineno ()` and `get_driver_lineno ()`

By the way, I don't understand how driver line must increment twice (once in update_start_line and again in get_driver_lineno). I understand that we need to offset because numbering starts with 1 instead of 0, but why the second time?